src/drawer/pages/Ranking/Ranking.style.ts

```diff
@@ -0,0 +1,52 @@
+import styled from 'styled-components';
+
+export const StyledRankingContainer = styled.section`
+  display: flex;
+  flex-direction: column;
+  gap: 3rem;
+  margin-top: 3.37rem;
+  margin-bottom: 4rem;
+`;
+
+export const StyledCardContainer = styled.section`
+  display: grid;
+  grid-template-columns: 25.5rem 25.5rem 25.5rem;
```

Comment:
이렇게 같은 크기로 n등분 할 경우 `repeat`를 쓰면 더 깔끔한 코드가 될 거 같아요!!!!!

```suggestion
  grid-template-columns: repeat(3, 25.5rem);
```